tests/test_cli.py

```diff
@@ -101,11 +100,12 @@ def test_cli_early_exit(args):
         ),
     ],
 )
-def test_cli_path_mapping(
-    mock_load_json, mock_path, tmp_path, args, raises, expected
-):  # pylint: disable=unused-argument
+def test_cli_path_mapping(mock_load_json, mock_path, tmp_path, args, raises, expected):  # pylint: disable=unused-argument
```

Comment:
This feels like a long line for `ruff` - curious why it allows it here.

Reply:
#noqa/pylint codes seem to be allowed to break line length restricitions. i dont hate it.